src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountCollection.java

```diff
@@ -64,12 +65,34 @@ public void add(final SimpleInterval interval, final int refReadCount, final int
         counts.add(new AllelicCount(interval, refReadCount, altReadCount));
     }
 
+    /**
+     * Adds (interval, reference count, alternate count, reference nucleotide, alternate nucleotide) to respective lists.
+     * @param interval      site in 1-based format
+     * @param refReadCount  number of reads at site matching the reference
+     * @param altReadCount  number of reads at site different from the reference
+     * @param refNucleotide the ref nucleotide
+     * @param altNucleotide the alt nucleotide
+     */
+    public void add(final SimpleInterval interval, final int refReadCount, final int altReadCount,
```

Comment:
I think is better to substitute this method and the other overload into one that accepts ```AllelicCount``` instances; then is the using code's responsibility to instantiate the ```AllelicCount``` instance using the corresponding constructor.  

The reason this is better, is the decouples this classes code with changes in ```AllelicCount``` constructor signatures.